src/project_metadata.py

```diff
@@ -0,0 +1,21 @@
+import neptune.new as neptune
+from neptune.new.types import File
+import pandas as pd
+import plotly.express as px
+
+project = neptune.init_project()
+
+data = '../data/covid_and_healthy_spectra.csv'
+
+df = pd.read_csv(data)
+
+# Upload data
+project["data/files"].upload(data)
```

Comment:
This is a large file and rendering it slows down the browser. Use `track_files()` instead

Reply:
For future reference, what is the definition of large file? Cause this one is around 6 MB.

Reply:
Addressed.